src/main/java/seedu/address/ui/StrategyPanel.java

```diff
@@ -63,7 +72,20 @@ public StrategyPanel() {
         initCircle(player3, 30, 200, 200, Color.YELLOW);
     }
 
+    /**
+     * Changes the image contained in ImageView.
+     * @param file the file reference for the image to be loaded
+     */
+    public void changeImageBackground(File file) {
+        strategyImage.setImage((new Image((file.toURI().toString()))));
+    }
+
     private void initCircle(Circle circle, double rad, double x, double y, Paint color) {
+        strategyImage.setPreserveRatio(false);
+        strategyImage.fitWidthProperty().bind(strategyAnchorPane.widthProperty());
+        strategyImage.fitHeightProperty().bind(strategyAnchorPane.heightProperty());
+        strategyImage.setManaged(false);
+        strategyImage.toBack();
```

Comment:
These seem like initiator methods for Image? So I guess they should be put in the constructor of this class?

Reply:
Thanks for letting me know 👍.